devnet-sdk/systemext/l1.go

```diff
@@ -0,0 +1,64 @@
+package systemext
+
+import (
+	"github.com/ethereum-optimism/optimism/devnet-sdk/system2"
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+func WithL1(id system2.L1NetworkID, nodes []DefaultSystemExtL1NodeIDs) system2.Option {
+	return func(setup *system2.Setup) {
+		env := getOrchestrator(setup).env
+
+		commonConfig := setup.CommonConfig()
+		l1ID := eth.ChainIDFromBig(env.L1.Config.ChainID)
+		l1 := system2.NewL1Network(system2.L1NetworkConfig{
+			NetworkConfig: system2.NetworkConfig{
+				CommonConfig: commonConfig,
+				ChainConfig:  env.L1.Config,
+			},
+			ID: system2.L1NetworkID{
+				Key:     env.L1.Name,
+				ChainID: l1ID,
+			},
+		})
+
+		for idx, node := range env.L1.Nodes {
+			ids := nodes[idx]
+
+			elRPC, err := findProtocolService(setup, ELServiceName, RPCProtocol, node.Services)
+			require.NoError(setup.T, err)
```

Comment:
```suggestion
			setup.Require.NoError(err)
```
Nit (same with all other require usage).